x/profiles/types/msgs_app_links.go

```diff
@@ -0,0 +1,130 @@
+package types
+
+import (
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	clienttypes "github.com/cosmos/cosmos-sdk/x/ibc/core/02-client/types"
+	host "github.com/cosmos/cosmos-sdk/x/ibc/core/24-host"
+)
+
+const (
+	TypeMsgLinkApplication   = "link-application"
+	TypeMsgUnlinkApplication = "unlink-application"
```

Comment:
I think they should be put into keys.go, renamed into `ActionLinkApplication` and `ActionUnlinkApplication` with value of snake case.